unidist/cli/base/runner.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2021 Modin authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""``BackendRunner`` class functionality."""
+
+import os
+import subprocess
+
+from unidist.cli.base.utils import Defaults
+
+
+class BackendRunner:
+    """
+    Base class for any unidist backend runner.
+
+    Parameters
+    ----------
+    **cli_kwargs : dict
+        Keyword arguments supported by unidist CLI.
+    """
+
+    def __init__(self, **cli_kwargs):
+        self.script = cli_kwargs.get("script", "")
+        self.script_args = cli_kwargs.get("script_args", [""])
+        self.module = cli_kwargs.get("module", None)
+        self.backend = cli_kwargs.get("backend", Defaults.BACKEND)
+        self.hosts = cli_kwargs.get("hosts", Defaults.HOSTS)
```

Comment:
Should base BackendRunner know about hosts and redis_password? They are only related to specific backends.

Reply:
No, `BackendRunner ` should initialize only general properties. Init of properties `hosts` and `redis_address` moved to backends which use those.